src/jsi/jsi.h

```diff
@@ -1260,6 +1260,13 @@ class JSI_EXPORT JSError : public JSIException {
     return stack_;
   }
 
+  /// In V8's case, creating an Error object in JS doesn't record the callstack
+  /// To preserve it, we need a way to manually add the stack here and on the JS side
+  void setStack(std::string stack) {
```

Comment:
>g  [](start = 26, length = 2)

this should probably be &&

Reply:
I agree that would be more clear (and we'd do a bind instead of a move for stack), but I'm trying to keep it consistent with the coding style in here, in case we can push this upstream (see constructor signatures above).

---
In reply to: [492188176](https://github.com/microsoft/v8-jsi/pull/28#discussion_r492188176) [](ancestors = 492188176)